test_scripts/API/RAI/ideas_RegisterAppInterface_parameter_steeringWheelLocation.txt

```diff
@@ -0,0 +1,58 @@
+./test_scripts/API/RAI/046_ATF_N_HMI_Does_Not_Respond_WARNINGS_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_UI_GetCapabilities_HMI_doesn't_respond_RAI_WARNINGS
+./test_scripts/API/RAI/047_ATF_N_HMI_Does_Not_Respond_RESUME_FAILED_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_UI_GetCapabilities_HMI_doesn't_respond_RAI_RESUMED_FAILED
+./test_scripts/API/RAI/048_ATF_N_HMI_Does_Not_Respond_WRONG_LANGUAGE_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_UI_GetCapabilities_HMI_doesn't_respond_RAI_WRONG_LANGUAGE
+./test_scripts/API/RAI/049_ATF_N_steeringWheelLocation_IsWrongDataType_WARNINGS_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_UI_GetCapabilities_wrong_data_type_RAI_WARNINGS
+./test_scripts/API/RAI/050_ATF_N_steeringWheelLocation_IsWrongDataType_RESUME_FAILED_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_UI_GetCapabilities_wrong_data_type_RAI_RESUME_FAILED
+./test_scripts/API/RAI/051_ATF_N_steeringWheelLocation_IsWrongDataType_WRONG_LANGUAGE_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_UI_GetCapabilities_wrong_data_type_RAI_WRONG_LANGUAGE
+./test_scripts/API/RAI/052_ATF_P_steeringWheelLocation_LEFT_WARNINGS_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_UI_GetCapabilities_left_RAI_WARNINGS
+./test_scripts/API/RAI/053_ATF_P_steeringWheelLocation_LEFT_RESUME_FAILED_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_UI_GetCapabilities_left_RAI_RESUME_FAILED
+./test_scripts/API/RAI/054_ATF_P_steeringWheelLocation_LEFT_WRONG_LANGUAGE_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_UI_GetCapabilities_left_RAI_WRONG_LANGUAGE
+./test_scripts/API/RAI/055_ATF_P_steeringWheelLocation_CENTER_response_transfered_fake_param_WARNINGS_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_UI_GetCapabilities_center_fake_param_WARNINGS
+./test_scripts/API/RAI/056_ATF_P_steeringWheelLocation_CENTER_response_transfered_fake_param_RESUME_FAILED_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_UI_GetCapabilities_center_fake_param_RESUME_FAILED
+./test_scripts/API/RAI/057_ATF_P_steeringWheelLocation_CENTER_response_transfered_fake_param_WRONG_LANGUAGE_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_UI_GetCapabilities_center_fake_param_WRONG_LANGUAGE
+./test_scripts/API/RAI/058_ATF_P_steeringWheelLocation_LEFT_Resumption_RAI_steeringWheelLocation Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_UI_GetCapabilities_after_resumption
```

Comment:
@aderiabin, the idea is that SDL will keep the value of steeringWheelLocation(saved from UI_GetCapabilities) when mobile is disconnected and connected again during one ignition cycle.